csa/csa.go

```diff
@@ -0,0 +1,111 @@
+package csa
+
+import (
+	"fmt"
+	"github.com/RoaringBitmap/roaring"
+)
+const err = -1
+
+type Csa struct {
```

Comment:
public methods:
1. create CSA from text
2. find 1st occurence of a string, if any, with position
3. find all occurences of a string
4. get number of occurences of a string